### PR TITLE
msiz790-v1.1.2: initial commit

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+the filesystem and flash BIOS SPI under certain circumstances and have access
+to a specific USB port. That microcontroller does not run open-source firmware
+yet and could be leveraged by attackers with the ability to update it or have
+physical access. The only reasonable protection for Intel platforms is Intel
+Boot Guard with owner keys fused into the chipset, which we deliver as part of
+our Dasharo Enterprise offering.
+
+### Raptor Lake-S CPU Support
+
+We've started to add support for the upcoming Raptor Lake-S CPUs in Dasharo.
```
Comment (miczyg1) [inline blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md:107]:
```suggestion
We've added support for the Raptor Lake-S CPUs in Dasharo.
```

We finished, not started. Raptor Lake-S was upcoming a year ago (release date was 20th October 2022), now the Raptor Lake-S Refresh is upcoming.

File: blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+### UEFI 2.8 Errata C Compliance in EDKII Fork
+
+![UEFI Logo](/img/368px-Logo_of_the_UEFI_Forum.svg.png)
+
+In the v1.1.2 release, we have updated Dasharo to be compliant with the UEFI
+2.8 Errata C specification, released in January 2021, a standard that dictates
+how the firmware communicates with the operating system.
+
+This transition brings a notable enhancement to the firmware's cryptographic
+system, introducing support for the SHA-3 hash algorithm, known for
```
Comment (miczyg1) [inline blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md:149]: I don't see SHA-3 anywhere in the following:
https://github.com/Dasharo/edk2/pull/62
https://github.com/Dasharo/edk2/pull/63
https://github.com/Dasharo/edk2/pull/64
https://github.com/Dasharo/edk2/pull/65

Where does it come from?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+{{< youtube iTkXunUAriE >}}
+
+Of course, that means some microcontroller sitting on the mainboard can read
+the filesystem and flash BIOS SPI under certain circumstances and have access
+to a specific USB port. That microcontroller does not run open-source firmware
+yet and could be leveraged by attackers with the ability to update it or have
+physical access. The only reasonable protection for Intel platforms is Intel
+Boot Guard with owner keys fused into the chipset, which we deliver as part of
+our Dasharo Enterprise offering.
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md]: @miczyg1 fixed https://github.com/3mdeb/news-and-ideas/pull/537/commits/fd2d7b1d4890c1c92b103a222c7936afc90a655b

File: blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md (hunk 2 above)
Comment (pietrushnic) [reply blog/content/post/2023-09-08-msi_z690-a_v1.1.2_release.md]: @miczyg1 you are right that was plain wrong, I did analysis of the MRs you pointed and completely rewrote this section: https://github.com/3mdeb/news-and-ideas/pull/537/commits/87b20686469991eaa52f7b41b1680bea00288344

**Review by miczyg1 — APPROVED** [review]

**Comment by miczyg1** [comment]
@artur-rs can you help @pietrushnic with the SEO linkchecker? Something is off and I can't track it down since none of the error actually are related to the changes made in PR... So I wonder what is going on since previous checks are green....

**Comment by pietrushnic** [comment]
@miczyg1 I doubt @artur-rs can do anything with that. @Asiderr promised to fix that today. The problem is related to the fact that we check external links and the internet is in constant change, e.g. LPC website certs expired and there is redirection to other domain.

**Comment by miczyg1** [comment]
> LPC website certs expired and there is redirection to other domain.

Really? https://github.com/3mdeb/news-and-ideas/pull/540

**Comment by pietrushnic** [comment]
@Asiderr question is if it is possible to improve CI?

**Pull request merged**